crates/rolldown_testing_config/src/lib.rs

```diff
@@ -22,6 +22,9 @@ pub struct TestConfig {
   #[serde(default)]
   /// If `true`, the sourcemap visualizer will be snapshot.
   pub visualize_sourcemap: bool,
+  #[serde(default)]
+  /// If `true`, the fixture throw an unknown error that can be caught in the final execution phase.
+  pub expect_uncaught_error: bool,
```

Comment:
I don't really quite get the idea of `expect_uncaught_error`? If some fixtures panics, that means there are problems in rolldown's code and need to be fixed. Marking it with `expect_uncaught_error: true` helps nothing.

Reply:
Currently, the case parsing for the two packagejsons are not included in vec[buildError], esbuild is trying to test that if there is any writing error in the packagejsons, the program can be terminated normally and throw an exception, so far there is no problem with the implementation of rolldown. However, if you want to continue testing in all cases without interruption, you need to catch those errors that are not in the context of vec[buildError].

Reply:
Based on these issues, so I added these codes